crates/dpapi/src/error.rs

```diff
@@ -2,16 +2,16 @@ use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum Error {
-    #[error("{0:?}")]
+    #[error("IO error")]
     Io(#[from] std::io::Error),
 
-    #[error("UUID error: {0:?}")]
+    #[error("UUID error")]
     Uuid(#[from] uuid::Error),
 
-    #[error("integer conversion error: {0:?}")]
+    #[error("integer conversion error")]
     IntConversion(#[from] std::num::TryFromIntError),
```

Comment:
note: It also occurred to me that `thiserror` has a `error(transparent)` attribute for situations where adding an additional message is not relevant.

https://docs.rs/thiserror/latest/thiserror/#details

Also a worthwhile read: https://www.howtocodeit.com/articles/the-definitive-guide-to-rust-error-handling